PhilosophicalIssues.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Please add the content that should go into these page. Not just an empty html page.